google/gax/api_callable.py

```diff
@@ -30,107 +30,16 @@
 """Provides function wrappers that implement page streaming and retrying."""
 
 from __future__ import absolute_import, division
-import random
-import time
 
 from future import utils
 
-from . import (BackoffSettings, BundleOptions, bundling, _CallSettings, config,
-               errors, PageIterator, ResourceIterator, RetryOptions)
-from .errors import RetryError
+from google.gax import (BackoffSettings, BundleOptions, bundling, _CallSettings,
+                        config, errors, PageIterator, ResourceIterator,
+                        RetryOptions, retry as rt)
```

Comment:
Use the same system that you use to solve a name collision with the stdlib: `retry_`.